chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -106,6 +110,10 @@
       done
     EOF
   end
+  # Deletes postgres data dir in non-HA scenario
+  directory "/var/lib/pgsql/data" do
+    action :delete
```

Comment:
@mjura If I understand correctly this is done very after the dump is done, but maybe is good idea to rename the directory instead of removing it, and remove it one the restore was done.